sros2/sros2/api/__init__.py

```diff
@@ -551,3 +552,55 @@ def generate_artifacts(keystore_path=None, identity_names=[], policy_files=[]):
             create_permissions_from_policy_element(
                 keystore_path, identity_name, policy_element)
     return True
+
+
+class PolicyBuilder(object):
+    """Load and modify an XML security policy."""
+    def __init__(self, policy_file_path):
```

Comment:
Maybe `def __init__(self, *, policy_file_path=None)` makes more sense, given that it's pretty optional?